packages/bathymetry/README.md

```diff
@@ -1,11 +1,11 @@
-# Bathymetry creation
```

Comment:
Have we run prettier against all the README.md files ( should we add a build step to do that? )

Reply:
Yeah, I have run prettier for all these Readme files. I think we should do a build step for it.